Libs/Core/ctkAbstractJob.h

```diff
@@ -55,7 +55,8 @@ class CTK_CORE_EXPORT ctkAbstractJob : public QObject
   Q_PROPERTY(QDateTime startDateTime READ startDateTime);
   Q_PROPERTY(QDateTime completionDateTime READ completionDateTime);
   Q_PROPERTY(QString runningThreadID READ runningThreadID WRITE setRunningThreadID);
-  Q_PROPERTY(QString loggedText READ loggedText WRITE setLoggedText);
+  Q_PROPERTY(QString loggedText READ loggedText WRITE addLoggedText);
```

Comment:
`addLoggedText` does not set the provided value, so it is not a setter - it would be better to remove it

```suggestion
  Q_PROPERTY(QString loggedText READ loggedText);
```